internal/filter/filter.go

```diff
@@ -29,8 +25,22 @@ func Match(str, query string) bool {
 		if matchString == "" {
 			matchString = match[2]
 		}
+		matchStrings[i] = matchString
+	}
+	return matchStrings
+}
 
-		if strings.Contains(strings.ToLower(str), strings.ToLower(matchString)) {
+// Split terms by spaces, except for spaces within quotes.
+// Like http://stackoverflow.com/questions/16261635/javascript-split-string-by-space-but-ignore-space-in-quotes-notice-not-to-spli
+// except:
+// - Treat the query ["san fran] as having the single term ["san fran"] instead of ["san", "fran"]
+// - Removes the delimiting quotes.
+func MatchText(str string, matchStrings []string, query string) bool {
+	if query == "" {
```

Comment:
Let's get rid of this check.  Which should never get to it.  Also it allows us to remove the `query` variable from the entire stack.

Reply:
or maybe we can replace it with a check if `matchStrings` is an empty list/nil (should be equivalent to "we have no query values)"